jme3-terrain/src/main/resources/Common/MatDefs/Terrain/Modular/PBRTerrainUtils.glsllib

```diff
@@ -0,0 +1,170 @@
+#ifndef __PBR_TERRAIN_UTILS_MODULE__
```

Comment:
Please review the entire file again. Ensure the code is correctly formatted, extra whitespace is removed, and there's a single blank line between methods. Order is important when writing code; it makes it easier to review and find errors.

Reply:
yes I'll make sure I cleanup that file before this PR is finalized.